resources/definitions/lulzbot_base.def.json

```diff
@@ -0,0 +1,179 @@
+{
+    "name": "LulzBot",
+    "inherits": "fdmprinter",
+    "version": 2,
+    "metadata":
+    {
+        "author": "FAME3D Base",
+        "file_formats": "text/x-gcode",
+        "firmware_last_version": "1.1.9.34",
+        "first_start_actions":["MachineSettingsAction"],
+        "has_machine_quality": true,
+        "has_materials": true,
+        "has_variants":true,
+        "machine_extruder_trains": {
+            "0": "lulzbot_base_285_extruder"
+        },
+        "manufacturer": "LulzBot",
+        "preferred_variant_name": "LulzBot SE",
+        "preferred_material":"generic_pla",
+        "supported_actions": ["MachineSettingsAction"],
+        "variants_name": "Tool Head",
+        "visible": false
+    },
+    "overrides":
+    {
+        "infill_before_walls":
+        {
+            "default_value": "False"
+        },
+        "layer_height_0":
+        {
+            "maximum_value_warning": "0.5"
+        },
+        "material_print_temperature":
+        {
+            "maximum_value_warning": "295",
+            "description": "The temperature used for printing. Set at 0 to pre-heat the printer manually.",
+            "minimum_value": "0",
+            "enabled": true,
+            "unit": "\u00b0C"
+        },
+        "machine_nozzle_size":
+        {
+            "default_value": 0.5
+        },
+        "material_diameter": { "default_value": 2.85 }
+    },
+   
+    "settings":
+    {
+                "speed":
+        {
+            "label": "Speed",
+            "icon": "category_speed",
+            "description": "Speed",
+            "type": "category",
+            "children":
+                        {
+                                 "acceleration_enabled":
```

Comment:
acceleration_enabled is already a setting in fdmprinter.def.json. Why re-define that here?